src/sass/main.scss

```diff
@@ -0,0 +1,4 @@
+@import "utils/variables";
+@import "utils/normilize";
```

Comment:
looks like there is a typo. should be 'normalize' instead of 'normilize'. rename this file.

Reply:
ok, thank you!

Reply:
you rename file normilize.scss to normalize.scss it's good but you also should name when you import this file in main.scss.
Second line of this file: '@import "utils/normilize"' -> you should also change name to normalize in this line.

Reply:
Oh, I changed this import, but didn't commit it. Thank you for review!